crates/symbolicator/src/services/symbolication.rs

```diff
@@ -2261,6 +2539,11 @@ impl SymbolicationActor {
                     NewStackwalkingProblem::Diff(_) => "Different stackwalking results",
                     NewStackwalkingProblem::Slow => "Slow stackwalking run",
                 };
+
+                if let NewStackwalkingProblem::Diff(ref diff) = problem {
+                    tracing::debug!("{}", diff);
+                }
+
```

Comment:
This part is obviously not necessary for the PR, but I think it doesn't hurt and helps local development.